src/configuration.c

```diff
@@ -543,6 +544,21 @@ config_param_val_long(char *str, long *dst, int non_negative)
 	return (1);
 }
 
+static int
+config_param_val_workers(char *str, long *dst, int non_negative)
+{
+	assert(str != NULL);
+
+	if (strcasecmp(str, "auto") == 0) {
+		/* Get number of active CPU */
+		*dst = get_nprocs_conf();
```

Comment:
I'd rather we added a configure check for `sysconf(_SC_NPROCESSORS_ONLN)` and use that if available, falling back to the default value of 1.